### PR TITLE
Add ssh to theia-dev image.

### PR DESCRIPTION
**Pull request by AndrienkoAleksandr**
### What does this PR do?
Apply ssh and ssh-generator to the Theia dev image.

### What issues does this PR fix or reference?
Internal use.

### CODE REVIEW

File: dockerfiles/theia-dev/Dockerfile [hunk 1]
```diff
@@ -23,6 +23,14 @@ RUN yarn global add @eclipse-che/theia-generator@0.0.1-1543478099
 # change permissions
 RUN find ${HOME} -exec sh -c "chgrp 0 {}; chmod g+rwX {}" \; 2>log.txt
 
+RUN apk update && \
+    apk add --no-cache \
+    openssh-keygen
+RUN apk add --no-cache openssh
+
+# Set up more file watcher
+RUN echo "fs.inotify.max_user_watches = 524288" > /etc/sysctl.conf
```
Comment (benoitf) [inline dockerfiles/theia-dev/Dockerfile]: let's move it after line 16 (so before installing something else) as this instruction may not change at all

**Comment by mmorhun** [comment]
I also face problem with watchers limit, so +1 to merge and apply as soon as possible

**Comment by l0rd** [comment]
@AndrienkoAleksandr @benoitf what is the purpose of adding openssh? How will it be used? What problem it solves?

**Comment by benoitf** [comment]
@l0rd cloning a repository with ssh

**Review by benoitf — APPROVED** [review]

**Pull request merged**